CONTRIBUTING.md

```diff
@@ -173,15 +173,15 @@ You will make changes to copies of thefiles which make up rambox in a personal f
 
 #### Important: ALWAYS EDIT ON A BRANCH
 
-Take away only one thing from this document: Never, **EVER** make edits to the `staging` branch.
+Take away only one thing from this document: Never, **EVER** make edits to the `master` branch.
 ALWAYS make a new branch BEFORE you edit files.
-This is critical, because if your PR is not accepted, your copy of staging will be forever sullied and the only way to fix it is to delete your fork and re-fork.
+This is critical, because your copy of `master` will be forever sullied and the only way to fix it is a difficult hard-reset and force-push process, or deleting your fork and re-forking.
```

Comment:
It's actually not necessary to re-fork - assuming you have `upstream` set up correctly (GitHub Desktop does it for you automatically), you can always fix master with:

```
git checkout master
git reset --hard upstream/master
git push -f
```

Reply:
If you don't have `upstream` set up correctly, you can just do:

```
git config -e
```

And then, above `[remote "origin"]`, add

```
[remote "upstream"]
	url = git@github.com:ramboxapp/community-edition.git
	fetch = +refs/heads/*:refs/remotes/origin/*
```

Reply:
If you don't have `upstream` set up correctly, you just need to do this before everything else:

```
git remote add upstream git@github.com:ramboxapp/community-edition.git
```